src/scss/components/checkbox/mixins/_hxCheckboxFacade.scss

```diff
@@ -18,29 +19,37 @@
 // ========== INVALID ==========
 @mixin __checkboxFacade--invalid($pseudo-state: null) {
   @if $pseudo-state == focus {
-    border-color: var(--hxCheckbox-invalid-bordercolor, $red-status-900);
+    border-color: $red-status-500;
+    border-color: var(--hxCheckbox-invalid-bordercolor, $red-status-500);
     box-shadow: $focus-glow-invalid;
   }
   @else if $pseudo-state == hover {
+    background-color: $red-status-100;
     background-color: var(--hxCheckbox-invalid-hover-bgcolor, $red-status-100);
```

Comment:
For the Required state. I agree that this is a rare use case, but is certainly possible in our BPiT forms within CTUI.
Double check with design and confirm the specs with them for the hover background color and the entire state in general. I assume it would just stay the same, but they would need to include this state in their documentation. Seems like it was missed in the design.

<img width="660" alt="Screen Shot 2021-03-11 at 11 02 24 AM" src="https://user-images.githubusercontent.com/14296817/110825548-c98b9380-8259-11eb-8099-09c17acc5c52.png">

Reply:
@danielmdesigns, can you provide design recommendations for `required` and `invalid` states.  These states could occur during validation phase (for example, form validation).